dbt/scripts/run_iasworld_data_tests.py

```diff
@@ -855,6 +905,10 @@ def main() -> None:
             ["run_year"],
         ),
     ]:
+        if not metadata_list:
+            print(f"{tablename} is empty, skipping metadata output")
+            continue
```

Comment:
This fixes a bug that I hadn't noticed until implementing the `--select` and `--selector` options: If a test run passes, pyarrow will fail to create a Parquet file from `test_run_failing_row`, since there's no `run_year` to partition on.